### PR TITLE
Guide

### PR DESCRIPTION
**Pull request by makoto**
- Added "Getting started" static page
- Added "Dropdown" with link to getting started
- Added a link from banner error page to getting started

NOTE: Changing error popup with lightbox will be done in different PR.

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

File: src/routes/GettingStarted.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        <p>
+          Everyone commits a small amount of ETH when they RSVP, which is
+          refunded after the event check-in. Any no-shows lose their ETH, which
+          can then be split amongst the attendees.
+        </p>
+
+        <h2>How to connect</h2>
+
+        <p>
+          To use our service, you have to access our site with Ethereum Wallet.
```
Comment (hiddentao) [inline src/routes/GettingStarted.js]: with _an_ Ethereum Wallet

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by hiddentao** [comment]
Please add `gettingstarted` and `faq` to the route maps

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by hiddentao** [comment]
There needs to be more space between Guides and Events - it needs to match the space between Events and  the sign-in button

**Comment by hiddentao** [comment]
@jefflau ^

**Comment by hiddentao** [comment]
Also, note that on mobile browsers the top nav bar looks ugly already - with the addition of Guides it will be even more congested. I'd like to solve this once and for all - perhaps a dropdown hamburger menu? @jefflau

**Comment by makoto** [comment]
> Also, note that on mobile browsers the top nav bar looks ugly already - with the addition of Guides it will be even more congested. I'd like to solve this once and for all - perhaps a dropdown hamburger menu? @jefflau

**Comment by makoto** [comment]
Good point about mobile styling. If it's just about merging Guides and Events into the Hamburger menu, I can give a try in this PR. If we also need to merge with "sign in" button that would require a bit more refactoring hence I prefer doing it in the separate PR

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by jefflau** [comment]
FYI this is incomplete, but wanted to push what I had so far over the weekend.

**Comment by hiddentao** [comment]
That's cool sofar. Here are is what I think we should end up with:

* Footer has links to events and guide  (at the moment it just has events). Footer may need cleaning up on mobile to make this look ok.
* On desktop, all pages should show Guide and Sign in (at present the homepage just shows Events)
* On mobile, all pages should show hamburger menu. Inside the menu, instead of showing Guide we _could_ show its subitems as top-level items. Menu item order should be from most frequently used to least frequently used: Sign in, Events,  Getting started, Organiser guide, FAQ

**Comment by jefflau** [comment]
> That's cool sofar. Here are is what I think we should end up with:
> 
> * Footer has links to events and guide  (at the moment it just has events). Footer may need cleaning up on mobile to make this look ok.
> * On desktop, all pages should show Guide and Sign in (at present the homepage just shows Events)
> * On mobile, all pages should show hamburger menu. Inside the menu, instead of showing Guide we _could_ show its subitems as top-level items. Menu item order should be from most frequently used to least frequently used: Sign in, Events,  Getting started, Organiser guide, FAQ

This mostly sounds fine except for the current design doesn't really cater for showing all menu items and the hamburger on the homepage. This should dealt with in a separate PR at the very least so I won't continue the discussion here. 

I think for mobile we can show the guides as subitems, but shown at all times just to keep the hierarchy.

**Comment by jefflau** [comment]
Hamburger is now ready for review @makoto @hiddentao

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by hiddentao** [comment]
@jefflau can see menu links overlapping page when closed. Also, why is the menu button only 2 lines and not 3?

screenshot 2019-02-14 at 09 20 48

**Comment by jefflau** [comment]
Fixed. Messed it up adding the transition. 

It's cleaner than 3 lines. We use 2 for ENS, and the inspiration is from apple. And it was quicker.

screenshot 2019-02-15 at 12 32 06

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

File: src/components/Header/Guide.js [hunk 2]
```diff
@@ -32,6 +32,16 @@ const Link = styled('a')`
...
+  { label: 'Getting Started', href: '/gettingstarted' },
+  { label: 'FAQ', href: '/faq' },
+  {
+    label: 'Event Organiser guide',
+    href:
+      'https://medium.com/wearekickback/kickback-event-organiser-guide-c2146c12defb'
+  }
+]
+
 export default class Dropdown extends Component {
```
Comment (hiddentao) [inline src/components/Header/Guide.js]: Please rename this component to `Guide`

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by makoto** [comment]
Could you guys try this branch locally? For some reason I am getting this error.

screenshot 2019-02-21 at 23 44 49

**Comment by hiddentao** [comment]
@makoto getting same error when i run locally

**Comment by jefflau** [comment]
It works for me - can someone else check out what's going on?

**Comment by hiddentao** [comment]
Everything works except "event organizer guide" link

File: src/components/Header/HamburgerMenu.js [hunk 3]
```diff
@@ -18,13 +18,21 @@ const HamburgerMenuContainer = styled('div')`
   }
 `
 
+function isExternal(url) {
+  return /^https/.test(url)
```
Comment (hiddentao) [inline src/components/Header/HamburgerMenu.js]: Should be `http`

File: src/components/Header/HamburgerMenu.js [hunk 4]
```diff
@@ -18,13 +18,21 @@ const HamburgerMenuContainer = styled('div')`
...
+
 function HamburgerMenu({ isMenuOpen }) {
   return (
     <HamburgerMenuContainer isMenuOpen={isMenuOpen}>
       <Link to="/events">Events</Link>
-      {links.map(l => (
-        <Link to={l.href}>{l.label}</Link>
-      ))}
+      {links.map(l =>
+        isExternal(l.url) ? (
```
Comment (hiddentao) [inline src/components/Header/HamburgerMenu.js]: Meant to be other way around? if external use `a` otherwise use `Link`?

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr78-app-wearekickback.surge.sh

**Pull request merged**